rio_cogeo/cogeo.py

```diff
@@ -660,7 +660,7 @@ def cog_info(
                 Bands=src_dst.count,
                 Width=src_dst.width,
                 Height=src_dst.height,
-                Tiled=src_dst.is_tiled,
+                Tiled=(src_dst.block_shapes[0][1] != src.width),
```

Comment:
Rasterio's `is_tiled` is kind of a mess: https://github.com/rasterio/rasterio/blob/main/rasterio/_base.pyx#L1011-L1018. The new code here is all rio-cogeo needs and nothing more.